src/spritekit.cs

```diff
@@ -376,6 +380,15 @@ partial interface SKEffectNode : SKWarpable {
 		[NullAllowed] // by default this property is null
 		[Export ("shader", ArgumentSemantic.Retain)]
 		SKShader Shader { get; set; }
+
+		[iOS (9,0), Mac(10,11)]
+		[Export ("valueForAttributeNamed:")]
+		[return: NullAllowed]
+		SKAttributeValue GetValue (string key);
+
+		[iOS (9,0), Mac(10,11)]
+		[Export ("setValue:forAttributeNamed:")]
+		void SetValue (SKAttributeValue value, string key);
```

Comment:
since we cannot remove the call from the base class (breaking change) there's no point in having additional code on the subclass when `XAMCORE_4_0` is not defined.